tests/test_abandon_command.rs

```diff
@@ -56,9 +56,10 @@ fn test_rebase_branch_with_merge() {
 
     let stdout = test_env.jj_cmd_success(&repo_path, &["abandon", "d"]);
     insta::assert_snapshot!(stdout, @r###"
-    Rebased 1 descendant commits onto parents of abandoned commits
+    Abandoned commit b7c62f28ed10 d
     Working copy now at: 11a2e10edf4e e
     Added 0 files, modified 0 files, removed 1 files
+    Rebased 1 descendant commits onto parents of abandoned commits
```

Comment:
I think this line fits pretty well right below "Abandoned commit b7c62f28ed10 d". Moving the line here also makes it inconsistent with `jj rebase`.

Reply:
Good catch.